front_matter/introduction.md

```diff
@@ -0,0 +1,39 @@
+# Accelerator Lattice Standard
+
+This standard is 
+an effort to create a standard to promote lattice information exchange for particle accelerators.
```

Comment:
Could we have this sentence on one line in the source file (I think one sentence per line is standard practice to display git diffs more easily)?
```suggestion
This standard is an effort to create a standard to promote lattice information exchange for particle accelerators.
```

Reply:
Changed.